### PR TITLE
fix: fix long as strings parsing

### PR DESCRIPTION
**Pull request by stepansergeevitch**
Fix integration tests issue where long is passed as a string by system engine.

### CODE REVIEW

File: rows.go [hunk 1]
```diff
@@ -178,7 +182,11 @@ func parseSingleValue(columnType string, val interface{}) (driver.Value, error)
 	case intType:
 		return int32(val.(float64)), nil
 	case longType:
-		return int64(val.(float64)), nil
+		// long values as passed as strings by system engine
+		if unpacked, ok := val.(float64); ok {
+			return int64(unpacked), nil
+		}
+		return strconv.ParseInt(val.(string), 10, 64)
```
Comment (ptiurin) [inline rows.go:189]: Could we add parameters here so it's obvious what those numbers mean, e.g. `strconv.ParseInt(val.(string), base=10, bitSize=64)`, if the language allows this.

**Review by ptiurin — APPROVED** [review]
Apart from that looks ok.

File: rows.go (hunk 1 above)
Comment (stepansergeevitch) [reply rows.go:189]: Go doesn't have keyword arguments. I will add comments

**Comment by stepansergeevitch** [comment]
Fixed test run: https://github.com/firebolt-db/firebolt-go-sdk/actions/runs/6393492225

**Pull request merged**